content/doc/developer/tutorial-adopt/replace-joda-time-with-java-8-time.adoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: developersection
+title: Replace joda-time
+---
+
+:modified-files: src/main/ src/test/
+:task-identifier: replace-joda-time-with-java-8-time
+:task-description: Replace joda-time
+
+= Replace joda-time with Java 8 time
```

Comment:
No, it is still an issue for consideration.  If the plugin was originally written for Java 8, it may include a Joda time reference, even if it is now compiled with Java 11.

Reply:
okay disregard in that case!